.github/workflows/main.yml

```diff
@@ -76,8 +76,6 @@ jobs:
       run: docker exec test_container /PrairieLearn/docker/lint_html.sh
     - name: Run the Python typechecker
       run: docker exec test_container /PrairieLearn/docker/typecheck_python.sh
-    - name: Run the JavaScript typechecker
-      run: docker exec test_container /PrairieLearn/docker/typecheck_js.sh
```

Comment:
Is this no longer needed because the transpilation step implicitly typechecks? I note we are running `yarn typecheck` as an explicit step in the PT CI so I don't quite understand why we don't need it here.

Reply:
It's necessary in PT because we use Babel to transpile files. Over here, we're using `tsc` to transpile, which implicitly does a type check.

This will get a little hairy once we have TS files in `apps/prairielearn/assets/scripts` - those are built with ESBuild which _doesn't_ perform type-checking. Even then, I think it's better to have type-checking be part of the build. Building/type-checking is not inexpensive in CI, so I'd rather avoid double-processing all files (once for `typecheck`, once for `build`).